databricks/koalas/frame.py

```diff
@@ -881,8 +881,47 @@ def unique(self):
         sdf = self._sdf
         return DataFrame(spark.DataFrame(sdf._jdf.distinct(), sdf.sql_ctx), self._metadata.copy())
 
-    @derived_from(pd.DataFrame)
     def drop(self, labels, axis=0, errors='raise'):
```

Comment:
cc @thunterdb why do we have "errors" parameter that seem unused?

Reply:
Seems like just wanted to match the signature (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html#pandas.DataFrame.drop) Let's remove.

Reply:
Removing it.